doc/modules/covariance.rst

```diff
@@ -18,6 +18,12 @@ a population's covariance matrix under various settings.
 We assume that the observations are independent and identically
 distributed (i.i.d.).
 
+* See :ref:`sphx_glr_auto_examples_covariance_plot_covariance_estimation.py` for
+  an example on how to fit :class:`ShrunkCovariance` , :class:`LedoitWolf` and
+  :class:`OAS` objects to data and for visualizing their performances in terms of
+  likelihood.
+
+
```

Comment:
```suggestion
```

Remove the blank line.

Reply:
@StefanieSenger : maybe I shall add `.. rubric:: Examples` before the note added at the end of the top paragraph "2.6 Covariance estimation"

`.. rubric:: Examples`

`* See :ref:`sphx_glr_auto_examples_covariance_plot_covariance_estimation.py` for
  an example on how to fit :class:`ShrunkCovariance` , :class:`LedoitWolf` and
  :class:`OAS` objects to data and for visualizing their performances in terms of
  likelihood.`
  
  I will try it. What do you think about it?